### DISCUSSION TITLE
How to have consistent typography styles between markdown content and the rest of the website?

### DISCUSSION THREAD

**Discussion by nemanjam** [tailwindlabs/tailwindcss#13026] [Help]
Lets say I have blog website and I want to have consistent typography styles between markdown content that uses `prose prose-slate` classes and other pages with my custom components. So if I make my own `<Heading>` component for `h1, h2...` tags and I want same styles from `prose prose-slate` I need to wrap `<div class="prose prose-slate"><h1>{text}</h1></div>` so that selectors from prose can match `h1` and other tags. This extra div of course isn't something I want in my `<Heading>` component. Other alternative is manually copying all styles from prose class which is ugly duplication as well.

How can I reuse styles from prose class in my components without having extra parent element?

**Comment by wongjn** [comment]
You could consider:
- Having a single `<div class="prose prose-slate">` parent element wrapping the markdown content.
- Adding the equivalent prose styles to the `<h1>` element directly with utility classes.
